lib/cosmos/top_level.rb

```diff
@@ -141,15 +145,65 @@ def self.add_to_search_path(path, front = true)
   # path or nil if the file could not be found. This allows for user configuration
   # files to override COSMOS data file defaults.
   def self.data_path(name)
+    # Check USERPATH
     filename = File.join(::Cosmos::USERPATH, 'config', 'data', name)
     return filename if File.exist? filename
 
+    # Check extensions
+    begin
+      Bundler.load.specs.each do |spec|
+        spec_name_split = spec.name.split('-')
+        if spec_name_split.length > 1 and spec_name_split[0] == 'cosmos'
```

Comment:
This applies throughout but obviously you're enforcing a naming convention of cosmos-<gem name>. That jives with the [rubygems guide](http://guides.rubygems.org/name-your-gem/) but we'll need to communicate that somewhere.